web/src/lib/components/admin-page/restore-dialoge.svelte

```diff
@@ -14,24 +14,14 @@
 	};
 </script>
 
-<div
-	class="border bg-immich-bg dark:bg-immich-dark-gray dark:border-immich-dark-gray p-4 shadow-sm w-[500px] max-w-[95vw] rounded-3xl py-8 dark:text-immich-dark-fg"
+<ConfirmDialogue
+	title="Restore User"
+	confirmText="Continue"
+	confirmColor="green"
+	on:confirm={restoreUser}
+	on:cancel
 >
-	<div
-		class="flex flex-col place-items-center place-content-center gap-4 px-4 text-immich-primary dark:text-immich-dark-primary"
-	>
-		<h1 class="text-2xl text-immich-primary dark:text-immich-dark-primary font-medium">
-			Restore User
-		</h1>
-	</div>
-	<div>
-		<p class="ml-4 text-md py-5 text-center">
-			{user.firstName}
-			{user.lastName} account will restored
-		</p>
-
-		<div class="flex w-full px-4 gap-4 mt-8">
-			<Button color="green" fullwidth on:click={restoreUser}>Confirm</Button>
-		</div>
-	</div>
-</div>
+	<svelte:fragment slot="prompt">
+		<p><b>{user.firstName} {user.lastName}</b>'s account will be restored.</p>
```

Comment:
```suggestion
		<p>The account for <b>{user.firstName} {user.lastName}</b> will be restored.</p>
```

Reply:
maybe we should use the email address instead? email addresses are unique, whereas first name + last name combinations aren't.

Reply:
Coming across someone who has the same first and last names as you is quite rare (maybe if you're named John Doe...), and I'm pretty sure having two guys like that on your instance will mostly never happen

Reply:
I'm not really sure that's the point. I'd suggest that within a family (where Immich is likely to be used) it isn't out of the question. The last names will be the same and it's not uncommon for children to have similar names to their father, or depending on the family a close relative may have the exact same first name.